torch/distributed/fsdp/api.py

```diff
@@ -46,12 +46,22 @@ class ShardingStrategy(Enum):
       :class:`DistributedDataParallel` API. For gradients, this strategy
       synchronizes them (via all-reduce) after the backward computation. The
       unsharded optimizer states are updated locally per rank.
+    - ``HYBRID_SHARD``: Apply ``FULL_SHARD`` within a node, and data parallelism across
+        nodes. This results in reduced communication volume as expensive all-gathers and
+        reduce-scatters are only done within a node, and can be more performant for medium
```

Comment:
```suggestion
        reduce-scatters are only done within a node, which can be more performant for medium-
```

(`medium-sized` with a hyphen)

Reply:
[Easy] I recommend changing before landing